src/layers.js

```diff
@@ -1,3 +1,40 @@
+export const COLOR_RAMP = [
```

Comment:
which color ramp is this? Should we rename?

Reply:
Ramps `COLOR_RAMP` and `BIOCLIMATIC_COLOR_RAMP` are not needed there, I messed this up while rebasing. I going to remove them, thanks :+1: